docs/deploy-cloudstate-cluster.adoc

```diff
@@ -0,0 +1,152 @@
+= Deploy Cloudstate Chat sample on Lightbend Cloudstate managed service
+Lightbend, 2020
+Version 2.0, 2020-07-06
+:description: Cloudstate Chat Application 
+:keywords: Cloudstate, stateful, serverless, chat-sample, tutorial, LBCS
+:sectnums:
+:toc:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+[deploy-cscluster-overview]
+== Overview
+
+You have built the 3 microservices composing the Chat application: 
+
+* `chat-ui` a stateless service serving the UI
+* `friends` a stateful service to manage the friend list of an user
+* `presence` a stateful service to maintain the online state of an user
+
+You have built the docker images of these services and published these images to your Docker registry. You have also run each of these images and tested that the service is in working order.
+
+If this is not the case, please review <<../README.adoc#cs-chat-sample-overview,Cloudstate Sample Chat Application>> to complete the prerequisites.
+
+We are now ready to deploy the Chat application, which we have chosen to deploy using Lightbend Cloudstate managed services (managed hosting). Compared to the alternative of deploying on your own Kubernetes cluster, Cloudstate managed services relieve the developer from the burden related to the infrastructure:
+
+* no need to provision hardware to setup (and maintain!) a Kubernetes cluster
+* simplify network configuration
+* simplify deployment
+
+
+== Prerequisites
+
+The main requirement is to have a Cloudstate account and some CLI tools to deploy the services on the Cloudstate platform.
+
+The steps are described in https://docs.lbcs.dev/getting-started/index.html[Getting started with Cloudstate]. We have actually already completed the most time consuming steps as we have already built our service. Here we will just need to setup the prerequisites pertaining to the deployment:
+
+
+[[signup-cloudstate-account]]
+=== Sign-up Cloudstate account
+
+To deploy your service on Lightbend Cloudstate host platform, you will need to sign up an account. With your Cloudstate account, you will be granted a Kubernetes cluster on the cloud. This cluster is managed by Lightbend Cloudstate platform. It is customized to host services created using the https://github.com/cloudstateio/cloudstate[Cloudstate library].
+
+Instructions: https://docs.lbcs.dev/getting-started/lightbend-account.html[Using a Cloudstate account]
+
+
+=== Install Cloudstate CLI
+
+`csctl` is the command line tool that you will use to deploy application on your Cloudstate cluster. Instructions: https://docs.lbcs.dev/getting-started/set-up-development-env.html#_cloudstate_cli[Install Cloudstate CLI]
+
+The `csctl` minimum version required is 0.0.9 which you can verify by:
```

Comment:
would anyone use 0.0.9 instead of the current 0.0.11?

Reply:
The current is 0.0.12.

Reply:
Thx, changed to 0.0.12